contrib/kind.sh

```diff
@@ -77,6 +78,7 @@ usage() {
     echo "-cl  | --ovn-loglevel-controller   Log config for ovn-controller DEFAULT: '-vconsole:info'."
     echo "-dl  | --ovn-loglevel-nbctld       Log config for nbctl daemon DEFAULT: '-vconsole:info'."
     echo "-ep  | --experimental-provider     Use an experimental OCI provider such as podman, instead of docker. DEFAULT: Disabled."
+    echo "-eb  | --egress-gw-bridge          The interface to be used as bridge for external gw traffic."
```

Comment:
why even give the option here of providing an interface name? We know kind deployments have eth0 as their first interface, isn't it predictable what the 2nd interface will be? Also, when you connect the docker containers to the second docker network, there is no way to name the interface they get in order to match what is passed in via this arg right?

Seems like they could just provide the argument, and the description could be something like "Uses an alternate interface for egress gateway traffic".

Reply:
ah yes, makes sense. The second interface is always eth1 , that's what we pass in CI in https://github.com/ovn-org/ovn-kubernetes/pull/2289/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R355 

Did not think about abstracting away the interface name, I'll make that a flag.